atm.js

```diff
@@ -1,21 +1,99 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+//var balances.checkingBalance = parseInt($(".balance").eq(0).html().replace("$", ""));
+// This not a DOM object; it is a number.  You are only saying =0.
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var balances = {
+  checkingBalance: 0,
+  savingsBalance: 0
+};
 
+var buttons = {
+  chkDepButton: $("input").eq(1),
+  chkWitButton: $("input").eq(2),
+  savDepButton: $("input").eq(4),
+  savWitButton: $("input").eq(5)
+};
```

Comment:
Can you think of how creating an `Account` constructor would help you refactor `var buttons`? Imagine if you passed in a DOM element to the constructor - how could you use that to reference its checking and savings buttons?